applications/GeoMechanicsApplication/tests/three_dimensional_piping/ProjectParameters.json

```diff
@@ -0,0 +1,192 @@
+            "Max_X",
+            "Soil"
+        ],
+        "body_domain_sub_model_part_list": [
+            "Soil",
+            "Pipe"
+        ],
+        "max_piping_iterations": 1000
+    },
+    "output_processes": {
+        "gid_output": [
+            {
+                "python_module": "gid_output_process",
+                "kratos_module": "KratosMultiphysics",
+                "process_name": "GiDOutputProcess",
+                "Parameters": {
+                    "model_part_name": "PorousDomain.porous_computational_model_part",
+                    "output_name": "Cube_10sS_1",
+                    "postprocess_parameters": {
+                        "result_file_configuration": {
+                            "gidpost_flags": {
+                                "WriteDeformedMeshFlag": "WriteUndeformed",
+                                "WriteConditionsFlag": "WriteElementsOnly",
+                                "GiDPostMode": "GiD_PostAscii",
+                                "MultiFileFlag": "SingleFile"
+                            },
+                            "file_label": "step",
+                            "output_control_type": "step",
+                            "output_interval": 1,
+                            "body_output": true,
+                            "node_output": false,
+                            "skin_output": false,
+                            "plane_output": [],
+                            "nodal_results": [
+                                "WATER_PRESSURE",
+                                "VOLUME_ACCELERATION",
+                                "HYDRAULIC_DISCHARGE"
+                            ],
+                            "gauss_point_results": [
+                                "FLUID_FLUX_VECTOR",
+                                "LOCAL_FLUID_FLUX_VECTOR",
+                                "LOCAL_PERMEABILITY_MATRIX",
+                                "HYDRAULIC_HEAD",
+                                "PIPE_ACTIVE",
+								"PIPE_HEIGHT"
+                            ]
+                        },
+                        "point_data_configuration": []
+                    }
+                },
+                "output_name": "Cube_10sS_1"
+            }
+        ]
+    },
+    "processes": {
+        "constraints_process_list": [
+            {
+                "python_module": "apply_scalar_constraint_table_process",
+                "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+                "process_name": "ApplyScalarConstraintTableProcess",
+                "Parameters": {
+                    "model_part_name": "PorousDomain.Max_X",
+                    "variable_name": "WATER_PRESSURE",
+                    "is_fixed": true,
+                    "fluid_pressure_type": "Hydrostatic",
+                    "gravity_direction": 2,
+                    "reference_coordinate": 0.0,
+                    "table": 0,
+                    "pressure_tension_cut_off": 10.0e6,
```

Comment:
Very peculiar value. Are we expecting a phreatic plane inside our model? Piping is probably only designed for full saturation, The retention curve is indeed SaturatedLaw.